x/stakers/keeper/logic_stakers.go

```diff
@@ -27,7 +26,7 @@ func (k Keeper) IncreaseStakerCommissionRewards(ctx sdk.Context, address string,
 	}
 
 	// transfer funds from pool to stakers module
-	if err := util.TransferFromModuleToModule(k.bankKeeper, ctx, poolTypes.ModuleName, types.ModuleName, amount); err != nil {
+	if err := k.bankKeeper.SendCoinsFromModuleToModule(ctx, poolTypes.ModuleName, types.ModuleName, amount); err != nil {
```

Comment:
We should probably also make the poolTypes.ModuleName an function argument (payerModuleName) to not assume any dependencies inside this function.
Similar to PayoutRewards in the delegation module